### PR TITLE
(SERVER-2859) Add docs for certificate clean endpoint

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by Magisus** [comment]
This should be held until the next 5.3.x release.

**Comment by Magisus** [comment]
@reidmv here are the docs for the `clean` endpoit.

File: documentation/http_certificate_clean.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+===============
+
+The `certificate clean` endpoint of the CA API allows the user to revoke and delete a list
+of certificates with a single request.
+
+```
+PUT /puppet-ca/v1/clean
+Content-Type: application/json
+```
+Takes a body with one required key, `certnames`, a list of certificates to clean. Each cert
```
Comment (K8med) [inline documentation/http_certificate_clean.markdown:16]: This first sentence seems incomplete?  Change to "It takes a body with one ...."

**Review by K8med — APPROVED** [review]
Just a small clarification needed, but otherwise it looks good!

**Comment by mwaggett** [comment]
This can be merged now, but @Magisus did you want to take Kate's suggestion?

**Comment by Magisus** [comment]
We'll also need to manually add this to the 6.x docs in the other repo too. I'll put something up.

**Review by mwaggett — APPROVED** [review]

**Pull request merged**